config/initializers/productization.rb

```diff
@@ -1 +1,3 @@
-Vmdb::Productization.new.prepare
+Rails.application.config.to_prepare do
```

Comment:
Yeah, setup paths once would be better...is something stomping on the paths once they are set?